internal/ui/src/components/empty.tsx

```diff
@@ -0,0 +1,68 @@
+import type React from "react";
+
+import { cn } from "../lib/utils";
+
+interface EmptyRootProps extends React.HTMLAttributes<HTMLDivElement> {
+  fill?: boolean;
```

Comment:
didn't we say we'd remove this?

https://unkey.slack.com/archives/C05SYU6P2DC/p1736285271083539